sjb/hack/determine_install_upgrade_version.py

```diff
@@ -114,7 +114,10 @@ def sort_pkgs(available_pkgs):
 	# listing of origin or docker pkgs will be disabled, based on the enabled excluder.
 	yb.conf.disable_excludes = ["all"]
 	sys.stdout = old_stdout
-	available_pkgs = rpmutils.unique(yb.doPackageLists('available', patterns=[pkg_name], showdups=True).available)
+
+	generic_holder = yb.doPackageLists('all', patterns=[pkg_name], showdups=True)
+	available_pkgs = rpmutils.unique(generic_holder.available + generic_holder.installed)
```

Comment:
not really sure what you mean... the thing is that the upgrade version should be the latest (if the `--dependency-branch` flag is not pointing to a release branch)